doc/user/data/explain_plan_operators.yml

```diff
@@ -218,6 +386,16 @@ operators:
     expansive: False
     example: "`ArrangeBy keys=[[#0]]`"
 
+  - operator: Arrange
```

Comment:
Was this supposed to be ArrangeBy? (See comment on line https://github.com/MaterializeInc/materialize/pull/31054/files#diff-8a8cb8a7b9f7777ed5d1f88369aa9a4c9ef159e56cd49f4f1a2e309435d707deR370). -- although, the example has Arrange 12 ... but the description talks about keys.

Reply:
No---I had it just output `Arrange`, in part because `mz_lir_mapping` won't show any info about the keys. (Corrected the text on this.)